ansible/library/switch_arptable.py

```diff
@@ -59,7 +102,7 @@ def main():
     if arp_tbl == None:
```

Comment:
based on the impl, it can never return None. It can only throw exception, you need to catch exception.